### PR TITLE
bpo-29660: traceback: Document that etype is ignored in some places.

### PR DESCRIPTION
**Pull request by Carreau**
See http://bugs.python.org/issue29660

### CODE REVIEW

File: Doc/library/traceback.rst [hunk 1]
```diff
@@ -45,16 +45,19 @@ The module defines the following functions:
...
+     indicating the approximate position of the error.
 
    The optional *limit* argument has the same meaning as for :func:`print_tb`.
    If *chain* is true (the default), then chained exceptions (the
    :attr:`__cause__` or :attr:`__context__` attributes of the exception) will be
    printed as well, like the interpreter itself does when printing an unhandled
    exception.
 
+   .. versionchanged:: 3.5
+      The *etype* argument is ignored and infered from the type of *value*.
```
Comment (vadmium) [inline Doc/library/traceback.rst]: inferred (also repeated below)

File: Doc/library/traceback.rst [hunk 2]
```diff
@@ -468,5 +474,5 @@ This last example demonstrates the final few formatting functions:
    ['  File "spam.py", line 3, in <module>\n    spam.eggs()\n',
     '  File "eggs.py", line 42, in eggs\n    return "bacon"\n']
    >>> an_error = IndexError('tuple index out of range')
-   >>> traceback.format_exception_only(type(an_error), an_error)
+   >>> traceback.format_exception_only("ignored parameter", an_error)
```
Comment (vadmium) [inline Doc/library/traceback.rst]: Does not seem to be the case for “format_exception_only”

File: Doc/library/traceback.rst [hunk 3]
```diff
@@ -372,7 +378,7 @@ exception and traceback:
        print("*** print_tb:")
        traceback.print_tb(exc_traceback, limit=1, file=sys.stdout)
        print("*** print_exception:")
-       traceback.print_exception(exc_type, exc_value, exc_traceback,
+       traceback.print_exception("ignored parameter", exc_value, exc_traceback,
```
Comment (vadmium) [inline Doc/library/traceback.rst]: This does not seem ideal. It is less compatible with older Python versions. If it needs clarifying at all, perhaps a comment would be better?

Comment (Carreau) [reply Doc/library/traceback.rst]: Yes, sure.

File: Doc/library/traceback.rst (hunk 1 above)
Comment (Carreau) [reply Doc/library/traceback.rst]: fixed.

File: Doc/library/traceback.rst (hunk 2 above)
Comment (Carreau) [reply Doc/library/traceback.rst]: indeed, fixed.

**Comment by Carreau** [comment]
Thanks, sorry for the delay to fix. Comments should be addressed.

File: Doc/library/traceback.rst [hunk 4]
```diff
@@ -372,6 +378,7 @@ exception and traceback:
        print("*** print_tb:")
        traceback.print_tb(exc_traceback, limit=1, file=sys.stdout)
        print("*** print_exception:")
+       # exc_type below ignored on 3.5 and later
```
Comment (Mariatta) [inline Doc/library/traceback.rst]: **is** ignored on 3.5 and later 

The same for below :)

**Review by Mariatta — APPROVED** [review]

**Comment by Mariatta** [comment]
Thanks @Carreau 🎉

**Pull request merged**